setup/gcp/installer/util/constants.py

```diff
@@ -35,6 +35,12 @@ class FirewallRuleDirection(Enum):
 DEFAULT_BUCKET_FMT_V1 = 'gs://{}-data-{}'
 DEFAULT_BUCKET_FMT_V2 = 'gs://{}-{}-data-{}'
 
+FORSETI_V1_RULE_FILES = [
+    'bigquery_rules.yaml', 'blacklist_rules.yaml', 'bucket_rules.yaml',
```

Comment:
Can you please make each of these in a separate line.  It will be easier to keep them sorted, and add/remove.

Reply:
Updated.